adminutils/adminutils.py

```diff
@@ -237,6 +262,9 @@ async def emote_steal(self, ctx, name: str, message_id: discord.Message, *roles:
             return
         except discord.HTTPException as e:
             await ctx.send(chat.error(_("An error occurred on adding an emoji: {}").format(e)))
+        else:
+            await ctx.send(_(f"{em} created with the name `{em.name}`."))
```

Comment:
f-strings are not gonna work with i18n, please use `_("{} string {}).format(var, var)`